dev-util/smartgit/smartgit-6.5.6.ebuild

```diff
@@ -23,7 +23,7 @@ RESTRICT="fetch"
 RDEPEND=">=virtual/jre-1.7:1.7"
```

Comment:
The application starts without any `hg` or `git` executable installed.
However, it relies on them to perform vcs operations.
Therefore i think that `git` and `hg` USE flags (maybe defaulted to enabled?) are the right choice.

Since I am not familiar with ebuild writing, I have just updated the most straightforward things. 
I'll look into adding USE flags if they are needed.